server/tools/build-keycloak.sh

```diff
@@ -63,6 +63,11 @@ cd /opt/jboss/keycloak/modules/system/layers/base/org/mariadb/jdbc/main
 curl -L http://central.maven.org/maven2/org/mariadb/jdbc/mariadb-java-client/$JDBC_MARIADB_VERSION/mariadb-java-client-$JDBC_MARIADB_VERSION.jar > mariadb-jdbc.jar
 cp /opt/jboss/tools/databases/mariadb/module.xml .
 
+mkdir -p /opt/jboss/keycloak/modules/system/layers/keycloak/com/microsoft/sqlserver/jdbc/main
+cd /opt/jboss/keycloak/modules/system/layers/keycloak/com/microsoft/sqlserver/jdbc/main
+curl -L http://central.maven.org/maven2/com/microsoft/sqlserver/mssql-jdbc/$JDBC_MSSQL_VERSION.jre8/mssql-jdbc-$JDBC_MSSQL_VERSION.jre8.jar > mssql-jdbc.jar
```

Comment:
Isn't better to use only `$JDBC_MSSQL_VERSION` instead of `$JDBC_MSSQL_VERSION.jre8` ?